test/plugin_helper/data/cert/cert-key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Is this file used?

Reply:
No. Should I remove this? I put this file for future use due to the certificate file is always pair with the key file.

Reply:
I think no problem.

Reply:
My bad. no problem. 
I thought that other cert files already existed for the test. So this patch's cert files are only used for checking regex.

Reply:
No problem. Thank you for your review!